dips/dip-10.md

```diff
@@ -164,28 +157,48 @@ The format of the success response is:
     "status": "success",
     "result": {
 	    "_ObjectType": "ReferenceIDCommandResponse",
-	    "recipient_address": "c5ab123458df0003415689adbb47326d",
+	    "receiver_address": "dm1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4us2vfufk",
     },
     "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
 }
 ```
 
+The format of the failed response is:
+```
+{
+   "_ObjectType": "CommandResponseObject",
+    "status": "failure",
+    "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
+    "error": OffChainErrorObject(),
+}
+```
+
 **CommandResponseObject:**
 
 | Field 	    | Type 	     | Required? 	| Description 	           |
 |-------	    |------	     |-----------	|-------------	           |
 | _ObjectType   | str        | Y | Fixed value: `CommandResponseObject`|
 | status       | str | Y | Either `success` or `failure`. |
-| result       | Result object | Y | The Result obejct of response. |
+| result       | Result object | N | The Result obejct of response. |
 | cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
+| error          | [OffChainErrorObject](https://github.com/diem/dip/blob/main/dips/dip-1.mdx#offchainerrorobject) | N | Details of the error when status == "failure" |
+
 
 **CommandResultObject:**
 
 | Field 	    | Type 	     | Required? 	| Description 	           |
 |-------	    |------	     |-----------	|-------------	           |
 | _ObjectType   | str        | Y | Fixed value: `ReferenceIDCommandResponse`|
-| recipient_address       | str | Y | Receiver's on-chain address |
+| receiver_address       | str | Y | Receiver's onchain [account identifier](https://github.com/diem/dip/blob/main/dips/dip-5.md) with subaddress set to `None` or `00000000`|
+
 
+#### Error Codes
```

Comment:
can use invalid_field_value for invalid fields but we should be explicit what the cases are (I think that would have actually smoothened the convo in the preso today)